src/tabs/Artwork/components/ArtworkInfo.js

```diff
@@ -1,92 +1,107 @@
-import React, { useState, useEffect } from "react";
-import { Text, View, StyleSheet, Button, Image } from "react-native";
+import React, {useState, useEffect} from "react";
+import {Text, View, StyleSheet, Image} from "react-native";
 import axios from "axios";
-
+import Loading from '../../../shared/Loading';
 import ArtworkDetail from "zaila/src/tabs/Artwork/components/ArtworkDetail";
 
-const ArtworkInfo = ({sensorId}) => {
+const ArtworkInfo = ({route}) => {
+
+    const {sensorId} = route.params;
+
+    const [isReady,
+        setIsReady] = useState(false);
 
     const [artworkInfo,
         setArtworkInfo] = useState({
-        artworkId: 1,
-        exhibitionId: 123,
-        sensorId: "n123",
-        title: "The Pharao",
+        artworkId: null,
+        exhibitionId: null,
+        sensorId: "",
+        title: "Hello",
         imageURL: "https://www.somewhere.com/egypt_exhibition/1.pgg",
-        artistName: "Cleopatra",
-        media: "PNG",
-        year: "2020",
+        artistName: "",
+        media: "",
+        year: "",
         artworkDetails: [
             {
                 artworkDetailsId: 9,
                 artworkId: 1,
-                description: "The dog was the first animal to be domesticated, and has been selectively bred o" +
-                    "ver millennia for various behaviors, sensory capabilities, and physical attribut" +
-                    "es",
+                description: "",
                 languageCode: "en-US"
             }
         ]
     })
 
-  useEffect(() => {
-    const URL = `https://zaila-backend.herokuapp.com/api/artwork/1`;
-    axios
-      .get(URL, {
-        headers: {
-          "X-Custom-Header": "foobar"
-        }
-      })
-      .then(response => {
-        if (response.status === 200) {
-          setArtworkInfo(response.data[0].artwork);
-          // console.log(response.data[0].artwork)
-        }
-      })
-      .catch(err => {
-        console.log(err);
-      });
-  }, []);
+    useEffect(() => {
+
+        const receiveSensorId = sensorId
```

Comment:
Should this be `receiveSensorId == sensorId`?

Reply:
Sorry I don't get it. Why is double sign?

Reply:
The whole line is   `const receiveSensorId = sensorId? sensorId: 'n124';`

Reply:
All good! Finally spotted the purpose of this line.